src/App.js

```diff
@@ -1,23 +1,127 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+
+import babyNamesData from "./babyNamesData.json";
+import BabyNamesCard from "./BabyNamesCard";
+import SearchBar from "./SearchBar";
+import FavoriteNames from "./FavoriteNames";
+import ResetButton from "./ResetButton";
+import { useEffect, useState } from "react";
+import GenderSelector from "./GenderSelector";
 
 function App() {
+  let [favoriteName, setFavoriteName] = useState([]);
+  let [noneFavoriteNames, setNoneFavoriteNames] = useState(babyNamesData);
+  let [isClickingFavoriteNames] = useState(true);
+  const [showFavoriteNameMessage, setShowFavoriteNameMessage] = useState(true);
+  let storedFavoriteNames = localStorage.getItem("storedFavoriteNames");
+  let storedNoneFavoriteNames = localStorage.getItem("storedNoneFavoriteNames");
```

Comment:
I'm not sure why `localStorage` is being used to store names -- you are already storing the names in React state.

Reply:
the idea was to save favorites names on browser local storage and when the user comes back after closing the browser it will show the previous chosen name. I think it was one of the extra challenges.